perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -834,6 +842,71 @@ private Deserializer selectDeserializer(String format, HttpServletRequest req) t
 		}
 	}
 
+	private record OidcSubIssuer(String sub, String issuer) {}
+
+	private static OidcSubIssuer initOidcAuth(HttpServletRequest req, Map<String, String> additionalInformations) {
```

Comment:
very similar method as in the consolidatorManagerImpl - this method is retrieving actual oidc issuer and sub from proxy from user info, also in this method we are filling additionalInformation property with proper information
if additional information is not filled with proper information the checkForSimilarUser method will not work properly